undefined_syms.txt

```diff
@@ -683,6 +683,69 @@ D_060135EC = 0x060135EC;
 D_06013CD8 = 0x06013CD8;
 D_060140BC = 0x060140BC;
 
+// z_demo_ec
+D_06013B88 = 0x06013B88;
+D_060048F4 = 0x060048F4;
+D_0600BD38 = 0x0600BD38;
+D_0600B7B8 = 0x0600B7B8;
+D_06006C90 = 0x06006C90;
+D_0600196C = 0x0600196C;
+D_060000F0 = 0x060000F0;
+D_06005670 = 0x06005670;
+D_06005EA8 = 0x06005EA8;
+D_06007150 = 0x06007150;
+D_0600164C = 0x0600164C;
+D_06007958 = 0x06007958;
+D_0600164C = 0x0600164C;
+D_0600513C = 0x0600513C;
+D_06000330 = 0x06000330;
+D_06002254 = 0x06002254;
+D_0600BFA8 = 0x0600BFA8;
+D_0600C918 = 0x0600C918;
+D_060086D0 = 0x060086D0;
+D_06006C40 = 0x06006C40;
+D_06007FB8 = 0x06007FB8;
+D_060095DC = 0x060095DC;
+D_06002530 = 0x06002530;
+D_06009BB4 = 0x06009BB4;
+D_0600A07C = 0x0600A07C;
+D_0600A474 = 0x0600A474;
+D_06000CD8 = 0x06000CD8;
+D_06004258 = 0x06004258;
+D_0600AFE0 = 0x0600AFE0;
+D_06004658 = 0x06004658;
+D_0600AFE0 = 0x0600AFE0;
+D_060085F8 = 0x060085F8;
+D_06004868 = 0x06004868;
+D_0600FEF0 = 0x0600FEF0;
+D_06002FA0 = 0x06002FA0;
+D_06003A98 = 0x06003A98;
+D_06002FA0 = 0x06002FA0;
+D_060002B8 = 0x060002B8;
+D_06008D90 = 0x06008D90;
+D_06009EE0 = 0x06009EE0;
+D_06006930 = 0x06006930;
+D_06006220 = 0x06006220;
+D_06008D1C = 0x06008D1C;
+D_06009234 = 0x06009234;
+D_06004390 = 0x06004390;
+D_06004350 = 0x06004350;
+D_060076C0 = 0x060076C0;
+D_06007AC0 = 0x06007AC0;
+D_06005F20 = 0x06005F20;
+D_06006920 = 0x06006920;
+D_060004C8 = 0x060004C8;
+D_06005BD0 = 0x06005BD0;
+D_06005AC0 = 0x06005AC0;
+D_06005990 = 0x06005990;
+D_06005880 = 0x06005880;
+D_06009198 = 0x06009198;
+D_06009430 = 0x06009430;
+D_06009690 = 0x06009690;
+D_060035D8 = 0x060035D8;
+D_060074C8 = 0x060074C8;
+D_0600DE80 = 0x0600DE80;
+D_06003770 = 0x06003770;
```

Comment:
```suggestion
D_06003770 = 0x06003770;

```

Reply:
theres two sections for en_test anyway so i just removed the duplicate